data/sql/updates/pending_db_world/rev_1584982393046787000.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1584982393046787000');
+
+-- fix item_template missing spaces in description
+UPDATE `item_template` SET `description` = 'Counts as an Air, Earth, Fire, and Water totem.' WHERE `entry` IN (42593, 42594, 42595, 42596, 42601, 42606);
+
+-- deDE locale
+UPDATE `item_template_locale` SET `Description` = 'Zählt als Luft-, Erd-, Feuer- und Wassertotem.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'deDE';
+
+-- esES locale
+UPDATE `item_template_locale` SET `Description` = 'Cuenta como un tótem de aire, tierra, fuego y agua.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'esES';
+
+-- esMX locale
+UPDATE `item_template_locale` SET `Description` = 'Cuenta como un tótem de aire, tierra, fuego y agua.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'esMX';
+
+-- frFR locale
+UPDATE `item_template_locale` SET `Description` = 'Il compte comme un totem d''air, de terre, de feu et d''eau.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'frFR';
```

Comment:
```suggestion
UPDATE `item_template_locale` SET `Description` = 'Compte comme un totem d''air, de terre, de feu et d''eau.' WHERE `ID` IN (22345, 22395, 22396, 23005, 23199, 23200, 24413, 25645, 27544, 27815, 27947, 27984, 28066, 28248, 28357, 28523, 29389, 30023, 31031, 32330, 33078, 33505, 33506, 33507, 33843, 33939, 33940, 33941, 33951, 33952, 33953, 35104, 35105, 35106, 37575, 38361, 38367, 38368, 39728, 40267, 40322, 40708, 40709, 40710, 42593, 42594, 42595, 42596, 42597, 42598, 42599, 42601, 42602, 42603, 42604, 42606, 42607, 42608, 42609, 45114, 45169, 45255, 46978, 47665, 47666, 47667, 50458, 50463, 50464, 51501, 51507, 51513) AND `locale` = 'frFR';
```

seems more french to me but i don't know where it is supposed to show ingame